### PR TITLE
Add using configuration file instructions.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description
Add using configuration file instructions.

### CODE REVIEW

File: docs/HowTo/Configure/Use-Configuration-File.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+[environment variable](../../Reference/CLI/CLI-Syntax.md#teku-environment-variables). For options
+specified in more than one place, the order of precedence is command line, environment variable,
+configuration file.
+
+## YAML specification
+
+The configuration file must be a valid YAML file composed of key/value pairs. Each key is the same
+as the corresponding command line option name without the leading dashes (`--`).
+
+Values must conform to YAML specifications for string, numbers, arrays, and booleans. Specific
```
Comment (macfarla) [inline docs/HowTo/Configure/Use-Configuration-File.md:23]: strings?

File: docs/HowTo/Configure/Use-Configuration-File.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+differences between the command line and the TOML file format are:
```
Comment (macfarla) [inline docs/HowTo/Configure/Use-Configuration-File.md:24]:
```suggestion
differences between the command line and the YAML file format are:
```

File: docs/HowTo/Configure/Use-Configuration-File.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+description: Use Teku configuration file
+---
+
+# Using the Teku configuration file
+
+To specify command line options in a file, use a YAML configuration file.
+
+Save the configuration file and reuse it across node startups. To specify the configuration file,
```
Comment (macfarla) [inline docs/HowTo/Configure/Use-Configuration-File.md:9]:
```suggestion
To specify the configuration file,
```

Comment (macfarla) [reply docs/HowTo/Configure/Use-Configuration-File.md:9]: I don't think we need to spell that out

File: docs/HowTo/Configure/Use-Configuration-File.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+To override an option specified in the configuration file, either specify the same option on the
+command line or as an
+[environment variable](../../Reference/CLI/CLI-Syntax.md#teku-environment-variables). For options
+specified in more than one place, the order of precedence is command line, environment variable,
+configuration file.
+
+## YAML specification
+
+The configuration file must be a valid YAML file composed of key/value pairs. Each key is the same
+as the corresponding command line option name without the leading dashes (`--`).
```
Comment (macfarla) [inline docs/HowTo/Configure/Use-Configuration-File.md:21]: could simplify to - Each key is the corresponding command line option without the leading...

File: docs/HowTo/Configure/Use-Configuration-File.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+Values must conform to YAML specifications for string, numbers, arrays, and booleans. Specific
+differences between the command line and the TOML file format are:
+
+* Comma-separated lists on the command line are string arrays in the YAML file.
+* Enclose file paths, hexadecimal numbers, URLs, and &lt;host:port&gt; values in quotes.
```
Comment (macfarla) [inline docs/HowTo/Configure/Use-Configuration-File.md:27]:
```suggestion
* Enclose all string values (including but not limited to file paths, hexadecimal numbers, URLs, and &lt;host:port&gt;) in quotes.
```

Comment (macfarla) [reply docs/HowTo/Configure/Use-Configuration-File.md:27]: ^ this isn't a complete list eg network: "minimal" below
so I think you could actually remove the host:port bit from that list to make it neater

File: docs/HowTo/Configure/Use-Configuration-File.md (hunk 1 above)
Comment (bgravenorst) [reply docs/HowTo/Configure/Use-Configuration-File.md:23]: Done

File: docs/HowTo/Configure/Use-Configuration-File.md (hunk 3 above)
Comment (bgravenorst) [reply docs/HowTo/Configure/Use-Configuration-File.md:9]: Done

File: docs/HowTo/Configure/Use-Configuration-File.md (hunk 4 above)
Comment (bgravenorst) [reply docs/HowTo/Configure/Use-Configuration-File.md:21]: Done

File: docs/HowTo/Configure/Use-Configuration-File.md (hunk 5 above)
Comment (bgravenorst) [reply docs/HowTo/Configure/Use-Configuration-File.md:27]: Done

**Pull request merged**